src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
First, to be more precise we don't expose variables. We expose read-only objects that are extensible, but not overrideable.
And this is exactly the meaning of it.

A plugin can easily it's own constant to PluginEvents by doing:
`PluginEvents.add("MY_OWN_AUTOMATICALLY_PREFIX_UNIQUE_CONSTANT")`
After this point the contant isn't overrideble anymore.

This works aplication/plugins-wide.

I really don't see the advantage why this should be diverged into separate points.

How do you share PluginEvent-constants?